deploy/resources.go

```diff
@@ -86,7 +86,6 @@ spec:
     metadata:
       labels:
         name: camel-k-operator
-        app: "camel-k"
```

Comment:
Maybe this is out-of-sync. I've added a `make build-embed-resources` to sync that file. We should keep "camel-k" in the app label for the infrastructure components.

Reply:
Found it. It's the operator sdk that changes it. We can leave it and the source file without the label, as the file is not used (for this reason).